tests/test_actions.py

```diff
@@ -0,0 +1,86 @@
+import os
+import pygame
+import pytest
+from unittest.mock import MagicMock
+from flatland.actions.actions import MovementMixin, SpeechMixin, LimbControlMixin
+from flatland.consts import Direction
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from flatland.objects.base_objects import GameObject
+
+os.environ["SDL_VIDEODRIVER"] = "dummy"  # Use a headless display
+
+class Movement(MovementMixin):
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+        self.direction = None
+
+def test_move_up():
+    mover = Movement (0, 0)
```

Comment:
`mover = Movement(0, 0)`, please remove the space between the class and the init args.